extensions/data-workspace/src/dataworkspace.d.ts

```diff
@@ -146,4 +152,47 @@ declare module 'dataworkspace' {
 	export interface IProjectActionGroup {
 		actions: IProjectAction[];
 	}
+
+	/**
+	 * Defines project information to be presented in the dashboard container
+	 */
+	export interface IProjectInfo {
+		/**
+		 * name of the table
+		 */
+		tableName: string;
+
+		/**
+		 * column header information
+		 */
+		columnInfo: IDashboardColumnInfo[];
+
+		/**
+		 * data that goes in the table
+		 */
+		tableData: (IDashboardTableData | IDashboardTableDataGroup)[][];
+	}
+
+	/**
+	 * Header information of a column in the table
+	 */
+	export interface IDashboardColumnInfo {
+		displayName: string;
+		width: number;
+		valueType: azdata.DeclarativeDataType;
+	}
+
+	/**
+	 * List of data to be presented on the table in the dashboard container
+	 */
+	export interface IDashboardTableData {
+		value: any;
```

Comment:
shouldn't be any, right? also, you probably don't need this interface at all, you can just define a type for the possible values and directly use it.

Reply:
Updated it to string.